### DISCUSSION TITLE
关于ilogtail的熔断机制

### DISCUSSION THREAD

**Discussion by cyenxchen** [alibaba/ilogtail#937] [Help] [answered]
首先k8s本身会控制资源使用，那么如果ilogtial是运行在k8s环境中这个熔断机制是不是就没必要了？
另外，如何关闭这个熔断机制吗？

**Comment by messixukejia** [comment, marked as answer]
https://ilogtail.gitbook.io/ilogtail-docs/configuration/system-config

目前没有直接关闭的机制。可以通过调大cpu_usage_limit、mem_usage_limit达到关闭的效果。

如果可以帮忙提交个pr，在这个文档里说明些这个情况吧，多谢

**Reply by cyenxchen** [reply]
好的，已经提了。
